### PR TITLE
Added eSputnik

### PR DESCRIPTION
**Pull request by AlekseyDanchenko**
Hi @fstanis 
Could you please check this PR?

### CODE REVIEW

**Comment by AlekseyDanchenko** [comment]
@googlebot I signed it!

File: pages/shared/data/tools.yaml [hunk 1]
```diff
@@ -432,5 +432,16 @@ providers:
       width: 960
       height: 300
       fit: contain
+    formats:
+      - email
+  - name: eSputnik
+    url: https://esputnik.com/
+    type: Email Service
+    description: Supercharge Conversions of Your Bulk and Triggered Campaigns With Real-Time Recommendations In AMP Powered Emails
```
Comment (sebastianbenz) [inline pages/shared/data/tools.yaml]: One nit: please don't use title case

Comment (AlekseyDanchenko) [reply pages/shared/data/tools.yaml]: @sebastianbenz Done!

**Comment by fstanis** [comment]
Seems the CLA is still an issue, you need to sign it with the same email that you used when you made the commit (`git config user.email` to see it) or change the commit email to the one you signed the CLA with `git commit --amend --author="Your Name <someone@example.com>"`.

**Comment by AlekseyDanchenko** [comment]
@googlebot I signed it!

**Comment by AlekseyDanchenko** [comment]
@fstanis fixed CLA! :)

**Comment by fstanis** [comment]
Sorry, one small thing, could you use a smaller image perhaps (in kb and dimensions)? This one seems almost 100kb and that page has a lot of images on it, so we're trying to make sure it stays small.

Otherwise approved.

**Comment by AlekseyDanchenko** [comment]
@fstanis Done! )
This one is twice smaller

**Comment by AlekseyDanchenko** [comment]
@fstanis Filip, are we good to go? )

**Comment by sebastianbenz** [comment]
Thanks!

**Pull request merged**